packages/manager/core/api/src/iceberg.ts

```diff
@@ -55,20 +64,37 @@ function icebergFilter(comparator: FilterComparator, value: string | string[]) {
   }
 }
 
-export async function fetchIceberg<T>({
+export async function fetchIcebergV2<T>({
+  route,
+  pageSize,
+  cursor,
+}: IcebergFetchParamsV2): Promise<IcebergFetchResultV2<T>> {
+  const requestHeaders: Record<string, string> = {
+    'X-Pagination-Size': `${encodeURIComponent(pageSize || 5000)}`,
+    ...(cursor ? { 'X-Pagination-Cursor': `${cursor}` } : {}),
+  };
+
+  const { data, headers, status } = await apiClient.v2.get(route, {
+    headers: requestHeaders,
+  });
+
+  return { data, cursorNext: headers['x-pagination-cursor-next'], status };
+}
+
+export async function fetchIcebergV6<T>({
```

Comment:
if you rename fetchIceberg to fetchIcebergV6 you need to have a specific breaking change commit for this package

Reply:
Commit done  [here](bf17b9d6ae5658afedf16e4db547155181d9cb9e)